content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -17,7 +17,8 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - There's a new method **Get Customer** ([Web](/messaging/agent-chat-api/v3.2/#get-customer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-customer)) that returns info about a given Customer.
 - There's a new push [`chat_unfollowed`](/messaging/agent-chat-api/v3.2/rtm-reference/#chat_unfollowed) complementary to the **Unfollow Chat** method.
 - There's a new `include_chats_without_threads` filter in the  **List Chats** ([Web](/messaging/agent-chat-api/v3.2/#list-chats) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-chats)) method.
-- There's a new `agents` filter in the  **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It replaces the `agents_ids` filter. 
+- There's a new `agents` filter in the  **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It replaces the `agents_ids` filter.
+- There's a new pagination with hashed page IDs in the  **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It replaces old pagination.
```

Comment:
I think it'd be nice to note that the pagination is now uniform across whole API (if it is but I don't recall any method that uses other pagination format).

Reply:
We could try the following (format proposed by @gluszczyk in #656):
```
- the **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method:
  - a new `agents` filter replacing the `agents_ids` one
  - new pagination with hashed page IDs replacing the old pagination format. Pagination is now uniform across the whole Agent Chat API.
  -  a new `events.types` filter

Reply:
I think @litwinow is right, get_archives/list_archives was the last one with the old pagination format